datafusion/sqllogictest/test_files/spark/math/expm1.slt

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+query R
+SELECT spark_expm1(0::INT);
+----
+0
+
+query R
+SELECT spark_expm1(1::INT);
+----
+1.718281828459
```

Comment:
I tried running this in Spark 3.5.3 and did not get the same answer.

```
scala> spark.sql("select expm1(1)").show()
+-----------------+
|         EXPM1(1)|
+-----------------+
|1.718281828459045|
+-----------------+
```

Reply:
Does sqllogictest have a way to test floating point results within some tolerance?